addons/website/static/tests/tours/html_editor.js

```diff
@@ -6,6 +6,42 @@ import wTourUtils from "@website/js/tours/tour_utils";
 const adminCssModif = '#wrap {display: none;}';
 const demoCssModif = '// demo_edition';
 
+wTourUtils.registerWebsitePreviewTour('html_editor_language', {
+    url: '/test_page',
+    test: true,
+}, () => [
+    {
+        content: "open site menu",
+        extra_trigger: "iframe body:not(.editor_enable)",
+        trigger: 'button[data-menu-xmlid="website.menu_site"]',
+    }, {
+        content: "open html editor",
+        trigger: 'a[data-menu-xmlid="website.menu_ace_editor"]',
+    }, {
+        content: "add something in the page's default language version",
+        trigger: 'div.ace_line .ace_xml:contains("rommelpot")',
+        run: () => {
+            ace.edit(document.querySelector('#resource-editor div')).getSession().insert({
+                row: 1,
+                column: 1,
+            }, '<br/>\n');
+        },
+    }, {
+        content: "save the html editor",
+        extra_trigger: 'div.ace_line .ace_xml:contains("br")',
+        trigger: ".o_resource_editor .btn-primary",
+    }, {
+        content: "check that the page has the modification",
+        trigger: 'iframe #wrapwrap:has(br)',
```

Comment:
Seems fragile, there is good chance that  the page has a br inside it not related to your change and so the step test nothing (and if not, there is good chance it will be later), could you add something more unique maybe just add a class on the br if that's allowed?